src/elli_request.erl

```diff
@@ -174,8 +174,8 @@ query_str(#req{raw_path = Path}) ->
 
 
 %% @doc Parse the `Range' header from the request.
-%% The result is either a `byte_range_set()' or the atom `parse_error'.
-%% Use {@link elli_util:normalize_range/2} to get a validated, normalized range.
+%% The result is either a `[http_range()]' or the atom `parse_error'.
+%% Use `elli_util:normalize_range/2' to get a validated, normalized range.
```

Comment:
Will ex_doc make a link here?

Reply:
It should, yeah; it's pretty good at that. But lemme try it out, locally.

Reply:
<img width="573" alt="image" src="https://github.com/user-attachments/assets/1bffa417-5f0c-41b0-9c5c-5dfd1d2a4d62">